src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let operationCounter = 2;
+
+  const getsum = (firstNumber) => {
```

Comment:
You can return it without creating a variable.

```suggestion
  return (firstNumber) => {
```